eth/db/chain.py

```diff
@@ -395,6 +395,28 @@ def get_transaction_index(self, transaction_hash: Hash32) -> Tuple[BlockNumber,
         transaction_key = rlp.decode(encoded_key, sedes=TransactionKey)
         return (transaction_key.block_number, transaction_key.index)
 
+    def get_receipt_of_transaction(self,
```

Comment:
Should be `get_receipt_by_index` to mirror the naming convention established by `get_transaction_by_index`